lib/cc/cli/init.rb

```diff
@@ -75,11 +75,12 @@ def create_codeclimate_checks
       def create_default_engine_configs
         say "Generating default configuration for engines..."
         available_engine_configs.each do |(engine_name, config_paths)|
-          engine = engine_registry[engine_name]
-          config_mapping = Hash.new { |_, k| [k] }.merge(engine.fetch("config_files", {}))
+          if (engine = engine_registry[engine_name])
```

Comment:
This isn't ruby idiomatic and is an antipattern. Leave the assignment above. Depending on settings rubocop will probably bitch about it.